lib/kinu/http_client.rb

```diff
@@ -70,8 +70,8 @@ def original_filename
         File.basename(@file.path)
       end
 
-      def respond_to?(name)
-        io.respond_to?(name)
+      def respond_to?(name, include_all = false)
```

Comment:
The method should have 2 parameters because `Object#respond_to?` has them.

https://docs.ruby-lang.org/ja/latest/method/Object/i/respond_to=3f.html